lib/common/vm-common.js

```diff
@@ -195,6 +195,10 @@ function translateVm(obj, fullObject) {
     } else {
         vm.image_uuid = obj.image_uuid;
     }
+    // Consistently set VM tags to {} when empty:
+    if (vm.tags === undefined) {
```

Comment:
Might be better to use `if (!vm.tags) {` here - to catch the case where it's null as well.

Reply:
Sure, done. The reason to do not add it initially was to keep the same approach than the rest of that file ... but for the case of tags has sense to represent "null" as {} too.